pyramid_jsonapi/endpoints.py

```diff
@@ -53,7 +53,7 @@ def api_pattern(self, name, *components):
         """
         return self.pattern_from_components(
             '', self.main_prefix, self.api_prefix, name, *components
-        )
+        ).rstrip(self.sep)
```

Comment:
I moved this into this method, as it's part of the `RoutePatternConstructor` class, and I didn't think there would ever be a case where a trailing slash on a route would be useful/valid.

Pyramid does have a mechanism to allow trailing slashes - but it does so by re-requesting the route with the slash removed. Obviously this will not work for POST or similar requests -
 so is almost certainly not useful for this module.

https://docs.pylonsproject.org/projects/pyramid/en/latest/narr/urldispatch.html#redirecting-to-slash-appended-routes

Reply:
An alternative might be to have the method take an argument as to whether or not to strip the trailing separator (similar to what `strip` does)- if nothing else it's easier to discover and document, and means that the method caller down't need to also reference the instance `sep ` attribute.

Reply:
That sounds like a good idea.